extensions/twitch/general.py

```diff
@@ -12,30 +12,49 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
+
+import zoneinfo
+from datetime import datetime
+
 import asyncpg
 import discord
 import twitchio
-from twitchio.ext import commands
+from loguru import logger
```

Comment:
As per my comment in the PR, you don't need an external lib for logging since we already have one setup anyway.